src/Blog/Controller/Posts.php

```diff
@@ -50,23 +50,27 @@ public function run(): Response
 
     private function getPosts(): array
     {
-        return [
-            ["id" => "1", 
-            "title" => "Jak efektywnie nauczyć się programować?", 
-            "preamble" => "It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 'lorem ipsum' will uncover many web sites still in their infancy. Various versions have evolved over the years, sometimes by accident, sometimes on purpose (injected humour and the like).", 
-            "date" => "10-02-2020",
-            "image" => "/img/BlogPostHeader.jpg"], 
-            
-            ["id" => "2", "title" => "PHP - yay or nay?", 
-            "preamble" => "It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 'lorem ipsum' will uncover many web sites still in their infancy. Various versions have evolved over the years, sometimes by accident, sometimes on purpose (injected humour and the like).", 
-            "date" => "16-02-2020",
-            "image" => "https://my-test-11.slatic.net/shop/1240135492a9f2e557b7e2e7caebd097.png_1200x1200q75.jpg_.webp"], 
-            
-            ["id" =>"3", 
-            "title" => "PHP, SQL i inne bajery", 
-            "preamble" => "It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 'lorem ipsum' will uncover many web sites still in their infancy. Various versions have evolved over the years, sometimes by accident, sometimes on purpose (injected humour and the like).", 
-            "date" => "20-02-2020",
-            "image" => "https://image.freepik.com/foto-gratis/suministros-regreso-escuela_90258-1191.jpg"]
-        ];
+        $tablica=[];
```

Comment:
- Nazwa po polsku
- Nazwa powinna opisywać to co się pod nią kryje, niekoniecznie typ
- Formatowanie